app/assets/stylesheets/inline_checkboxes_override.scss

```diff
@@ -0,0 +1,5 @@
+.inline-checkbox {
+	display: inline-block;
+	width: 20%;
```

Comment:
This totally breaks on mobile, we should have some kind of breakpoint here and probably just increase them to full width.